app/models/user.rb

```diff
@@ -148,6 +149,16 @@ class User < ApplicationRecord
 
   scope :dev_account, -> { find_by(id: ApplicationConfig["DEVTO_USER_ID"]) }
 
+  scope :with_this_week_comments, lambda { |number|
```

Comment:
I included a few scopes as an example on how to use JSONB operators for queries

Reply:
From a file length perspective, how do you feel about maybe moving all scopes to a separate file one way or another? If we lean in here I could see this getting long.

Reply:
Oh I personally like the idea of at least counter scopes and even related methods being separate! We did something similar at Kenna with a model that had extensive status options and it worked out really well. Very easy to track down anything status-related bc it was all contained in a single file.

Reply:
I'm also down for organizing files in a sane way, some of our models are way too big!